### PR TITLE
Change layouts export to access list from layouts.all

### LINKED ISSUE #455: layouts.all.all instead of layouts.all?
I'm using version 6.1.0 within an app written in Coffeescript.

The documentation indicates that you can iterate through `maki.layouts.all` but in my experience `maki.layouts.all` is an object with a key `all`, so to iterate through the icons you need to access `maki.layouts.all.all`.

Is this something specific to my coffeescript setup, or does the documentation need to be updated?

```coffeescript
maki = require '@mapbox/maki'

maki.layouts.all # { all: [...] }
maki.layouts.all.all # [...]
```

### PR DESCRIPTION
**Pull request by tristen**
This results in a breaking change but is more right. It might be good to bundle this change along with icon updates.

---

Closes #455

### CODE REVIEW

**Comment by tristen** [comment]
> Also, even though I've approved, just want to check in whether this feels more right or removing the `all` key from `all.json` itself and leaving in `index.js` which I think feels more right to me

I think I'd need more intel on why it was set up like that in the first place. Was it future proofing to support more properties off the layout object?

**Comment by tristen** [comment]
@aparlato ok! I think I have enough to go on to make that change (looks like old piece left around when other data was exported). Back to you for review and thanks for flagging this 🙇

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The maki module exports `layouts` which is an object that can be used to organiz
 var maki = require('@mapbox/maki');
 
 files.forEach(function(fileName, j) {
-  maki.layouts.all.forEach(function(icon) {
+  maki.layouts.forEach(function(icon) {
```
Comment (aparlato) [inline README.md]: This won't work because layouts has the `all` key

**Comment by aparlato** [comment]
Also, looks like a test isn't passing

File: README.md (hunk 1 above)
Comment (tristen) [reply README.md]: Weird. I didn't commit the change to layout. Updated.

File: scripts/make-layouts.js [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
   fs.readdir('./icons/', function(err, files) {
     if (err) console.log(err);
 
-    var all = {
-      all: files
-        // Assume all icons include both size 11 and 15
-        .filter(file => file.indexOf('-15.svg') !== -1)
-        .map(file => file.split('-15.svg')[0])
-    };
+    const all = files
```
Comment (tristen) [inline scripts/make-layouts.js]: @aparlato this change should address what we were seeing 💫

**Pull request merged**